submissions/Conversee12/OOP Tiny-Js-World/index.js

```diff
@@ -0,0 +1,68 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Conversee12/a-tiny-JS-world.git
+   Web app: https://conversee12.github.io/a-tiny-JS-world/
+   */
+
+class Inhabitant {
+   constructor(species, name, gender, saying) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.saying = saying;
+   }
+   get tinyWorld() {
+      return `${this.species}; ${this.name}; ${this.gender}; ${this.saying}`;
```

Comment:
Please employ `Array` methods. Starting array may not necessarily consist of strings. Actually, anything can be an array's element.